### PR TITLE
Remove unused methods and use standard api

### PR DESCRIPTION
**Pull request by wreulicke**
Remove unused methods and deperecate methods in `Util`
And Use standard api.

----

Make sure these boxes are checked before submitting your PR -- thank you!

- [x] Added an entry into `CHANGELOG.md` if you have changed SpotBugs code

### CODE REVIEW

**Comment by ThrawnCA** [comment]
Removing deprecated methods should probably go into the 4.0 branch instead of master.

**Comment by wreulicke** [comment]
@ThrawnCA 

I think master branch is already released as 4.0.0-beta.
So, I selected master as base branch.
There is other branch?

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 ### Removed
 
 * non thread-safe implementation in `OpcodeStack.Item` ([#28](https://github.com/spotbugs/spotbugs/issues/28))
-* Remove unused methods in `BCELUtil` 
+* Remove unused methods in `BCELUtil`
+* Remove unused methods and deperecated methods in `edu.umd.cs.findbugs.util.Util`
```
Comment (KengoTODA) [inline CHANGELOG.md]: Move your entry into `Unreleased`

**Comment by KengoTODA** [comment]
Yes you're right, the `master` is appropriate branch for 4.0.0 development.

**Pull request merged**